react-native/screens/ProfileEditScreen.js

```diff
@@ -211,22 +245,45 @@ const ProfileEditScreen = props => {
   };
 
 
-    const getImage = () => {
-        return selectedImage;
-    };
-
-    const selectImage = imageData => {
-        setSelectedImage(imageData);
-    };
-
+  const getImage = () => {
+    return selectedImage;
+  };
 
+  const selectImage = imageData => {
+    setSelectedImage(imageData);
+  };
 
   const onKeyPress = (key) => {
     if (key === 'Enter') {
       descriptionInputRef.current.blur();
     }
   }
 
+  const renderDeleteDialog = () => {
+    return(
+      <Dialog
+        visible={isDialogVisible}
+        onTouchOutside={() => setIsDialogVisible(false)}
+        dialogTitle={<DialogTitle title="Are you sure you want to Delete your profile?"/>}
```

Comment:
Maybe this title should say 'account' instead of profile?

Or maybe "Are you sure you want to Delete your account and all associated postings? (This action is permanent)."

I know it's a verbose dialogue, but being very clear with user.

Reply:
Good point. Will make this change